charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -31,7 +31,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 72.9.1
+version: 73.0.0
```

Comment:
Can you bumb the kube version in the chart yaml and update the upgrade.md with the new minimum version?

Reply:
done